src/diffusion_for_multi_scale_molecular_dynamics/models/egnn.py

```diff
@@ -286,12 +287,12 @@ def __init__(
         coords_agg: str = "mean",
         message_agg: str = "mean",
         n_layers: int = 4,
-        num_classes: int = 2,
     ):
         """EGNN model stacking multiple E_GCL layers.
 
         Args:
             input_size: number of node features in the input
+            num_classes: number of atom types uses for the final node embedding.
```

Comment:
I understand the code as follows:
 - `num_atom_types` : the number of physical kinds of atoms to be treated: Si, Ge, Li, etc...
 - `num_classes`: the number of classes treated as one-hot.

We always have `num_classes = num_atom_types + 1`. 
Here, the meaning of `num_classes` is ambiguous.